util/session/sessionmanager.go

```diff
@@ -400,7 +401,13 @@ func (mgr *SessionManager) VerifyUsernamePassword(username string, password stri
 		return err
 	}
 	if !account.Enabled {
-		return status.Errorf(codes.Unauthenticated, accountDisabled, username)
+		log.Errorf(accountDisabled, username)
+		return InvalidLoginErr
```

Comment:
The previous implementation returns the error message that explains why login attempt failed. I think we should preserve that behavior.

Reply:
I was thinking of `return status.Errorf(codes.Unauthenticated, invalidLoginError, username)` here.